Formula/minia.rb

```diff
@@ -14,23 +14,27 @@ class Minia < Formula
   depends_on "cmake" => :build
   depends_on "zlib" unless OS.mac?
 
+  needs :cxx11
+
   def install
+    # Reduce memory usage for CircleCI.
+    ENV["MAKEFLAGS"] = "-j8" if ENV["CIRCLECI"]
```

Comment:
You can remove these two lines. The default is now four threads.

Reply:
Will we do a bulk removal from all recipes?

Reply:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

Reply:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

Reply:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

Reply:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.